common.mk

```diff
@@ -92,7 +92,7 @@ build-all: $(VERSIONS)
 
 .PHONY: $(VERSIONS)
 $(VERSIONS): % : %/root/help.1
-	VERSION="$@" $(script_env) $(build)
+	SINGLE_VERSION="$@" $(script_env) $(build)
```

Comment:
I doubt you want to do this as this would overwrite `SINGLE_VERSION` content the CI is providing to the Makefile scripts

Reply:
Solved by https://github.com/sclorg/container-common-scripts/pull/344/commits/19b3a5abf898cf6d1e7493276778f4773d0d383f

Reply:
Or rewrite common.mk script is needed.